app/javascript/dashboard/i18n/locale/en/inboxMgmt.json

```diff
@@ -47,7 +47,11 @@
       },
       "TWITTER": {
         "HELP": "To add your Twitter profile as a channel, you need to authenticate your Twitter Profile by clicking on 'Sign in with Twitter' ",
-        "ERROR_MESSAGE": "There was an error connecting to Twitter, please try again"
+        "ERROR_MESSAGE": "There was an error connecting to Twitter, please try again",
+        "TWEETS": {
+          "ENABLED": "Enable Tweets In Twitter Channel",
```

Comment:
We change the copy to "Create conversations from mentioned tweets". 
Also, rather than options, let's use a check box instead.